tests/test_optimization/test_conflict_detection_live.py

```
@@ -0,0 +1,336 @@
+                print(f"  [{i:3d}] delta.reasoning_content = ... (suppressing middle chunks)")
+
+    # stream_chunk_builder result
+    scb_response = litellm.stream_chunk_builder(chunks, messages=[{"role": "user", "content": prompt}])
+    scb_content = (
+        scb_response.choices[0].message.content
+        if scb_response and scb_response.choices
+        else None
+    )
+    assembled = "".join(content_parts)
+
+    print(f"\n--- SUMMARY ---")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)